qutebrowser/browser/commands.py

```diff
@@ -2234,7 +2234,16 @@ def fullscreen(self, leave=False):
             return
 
         window = self._tabbed_browser.widget.window()
-        window.setWindowState(window.windowState() ^ Qt.WindowFullScreen)
+        if window.isFullScreen():
```

Comment:
I think the `setWindowState` part of this if statement is the same as the xor before, right? I do like this more because it's a little more explicit though...

Reply:
The other option would be to do it like this:
```
        if not window.isFullScreen():
            window.state_before_fullscreen = window.windowState()
        window.setWindowState(window.windowState() ^ Qt.WindowFullScreen)
```
The behaviour should be the same.
I actually like the xor version a bit better as it separates the code for state_before_fullscreen and the actual fullscreen toggling. And it does it with less code. 
Should I switch to that one?

Reply:
It depends on what each person is familiar with. Personally, I think it's obvious that xor == toggle.

Reply:
@helbling I'm fine with either one, I was just a little thrown off why this part was changed at first. :)